app.py

```diff
@@ -19,16 +21,34 @@
 
 parser = reqparse.RequestParser()
 parser.add_argument('input')
+parser.add_argument('in_lang')
+parser.add_argument('out_lang')
 
 class Translate(Resource):
     def get(self):
         return {'work': 'ing'}
 
     def post(self):
+        # bring in post arguments
         args = parser.parse_args()
-        input_text = args['input']
-        gast = js_main.js_to_gast(input_text)
-        output_code = gast_to_py.gast_to_py(gast)
+        input_code = args['input']
+        input_lang = args['in_lang']
+        output_lang = args['out_lang']
+
+        # TODO: remove this once frontend is updated
```

Comment:
Even once the frontend is updated, requests shouldn't be assumed to be correct. This seems like a perfect situation to return an error code in the 400 range.